rpms/s3/buildrpm.sh

```diff
@@ -89,6 +89,7 @@ if ! [ -z "${GIT_VER}" ]; then
     if [ $ENABLE_DEBUG_LOG == 1 ]; then
         sed -i 's/#logLevel=DEBUG.*$/logLevel=DEBUG/g' auth/resources/authserver.properties.sample
         sed -i 's/S3_LOG_MODE:.*$/S3_LOG_MODE: DEBUG/g' s3config.release.yaml.sample
+        sed -i 's/file_log_level:.*$/file_log_level: 10/g' s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml.sample
```

Comment:
current default value in config file is already set to 10..can we update default value a bit less in config file itself ?
when we set  "ENABLE_DEBUG_LOG " flag true then we can go for log_level as 10

Reply:
@sachitanands , your view is right, but debug is the lowest value.Refer to the table below

Level | Numeric value
-- | --
CRITICAL | 50
ERROR | 40
WARNING | 30
INFO | 20
DEBUG | 10
NOTSET | 0

Reply:
can we set default bgdelete log level to "INFO" level and once we enable this flag we can set it to DEBUG simillar to Authserver.

Reply:
Agree to @sachinnerkar-art

Reply:
Changed in 731206dc1fd81dd1f8f3f4e09c6f314599d1b684